ui/event_handlers.go

```diff
@@ -570,3 +530,80 @@ func ImportStartedHandler(ctx context.Context, fr *frame.Frame, event partybus.E
 	}()
 	return err
 }
+
+// ShellOutputHandler takes bytes from a event.ShellOutput and publishes them to the frame.
+func ShellOutputHandler(ctx context.Context, fr *frame.Frame, event partybus.Event, wg *sync.WaitGroup) error {
+	reader, source, err := syftEventParsers.ParseShellOutputEvent(event)
+	if err != nil {
+		return fmt.Errorf("bad %s event: %w", event.Type, err)
+	}
+
+	titleLine, err := fr.Append()
+	if err != nil {
+		return err
+	}
+	wg.Add(1)
+
+	_, spinner := startProcess()
+	title := tileFormat.Sprintf("Running %s shell command...", source)
+	s := bufio.NewScanner(reader)
+	l := list.New()
+
+	formatFn := func(_ progress.Progress) {
+		spin := color.Magenta.Sprint(spinner.Next())
+		_, _ = io.WriteString(titleLine, fmt.Sprintf(statusTitleTemplate, spin, title))
+	}
+
+	var tlogEntry string
```

Comment:
Because we have to capture the t-log entry this isn't as generic as I would like it. This function will grow with matches for aux text if more shell out commands are used in syft. Probably fine for now, but needs a refactor in the future if we shell out again